### PR TITLE
target root directory

### PR DESCRIPTION
**Pull request by voyula**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
-.scrutinizer.yml    export-ignore
-.travis.yml         export-ignore
-CHANGELOG.md        export-ignore
-CONTRIBUTING.md     export-ignore
-docs                export-ignore
-phpunit.xml.dist    export-ignore
-README.md           export-ignore
-tests               export-ignore
+/.gitattributes      export-ignore
+/.gitignore          export-ignore
```
Comment (SammyK) [inline .gitattributes:6]: Let's keep the `.gitattributes` and `.gitignore` without the slash prefix so that they are also ignored in sub directories.

**Comment by SammyK** [comment]
@voyula Also - before I can merge this, can you sign the CLA linked above? Thanks! :)

**Comment by yguedidi** [comment]
@voyula same question than in #1018: any link that show the performance benefits? :)

**Comment by yguedidi** [comment]
Closed as no proof of performance gain, feel free to re-open a PR when you have some data

**Pull request closed without merging**